.changeset/grumpy-hats-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/eslint-plugin": major
+---
+
+Remove always/never from eslint rule options
```

Comment:
@drewkiimon these files are created by running `yarn changeset add` and describe how the version of a particular package should be bumped when published.